app/assets/javascripts/user-satisfaction-survey.js

```diff
@@ -81,6 +87,13 @@
 
       $surveyLink.attr('href', surveyUrl);
     },
+    inTestPeriod: function() {
```

Comment:
This function doesn't appear to be tested any longer - it is stubbed out in the tests. If that's the case, are we comfortable with that?